src/cdk/a11y/input-modality/input-modality-detector.ts

```diff
@@ -79,15 +79,16 @@ const modalityEventListenerOptions = normalizePassiveListenerOptions({
  * screen reader is akin to visually scanning a page, and should not be interpreted as actual user
  * input interaction.
  *
- * When a user is not navigating but *interacting* with a screen reader, this service's behavior is
- * largely undefined and depends on the events fired. For example, in VoiceOver, no keyboard events
- * are fired when performing an element's default action via Caps Lock + Space, thus no input
- * modality is detected.
+ * When a user is not navigating but *interacting* with a screen reader, this service attempts to
+ * update the input modality to keyboard.
  */
 @Injectable({ providedIn: 'root' })
 export class InputModalityDetector implements OnDestroy {
+  /** Emits whenever an input modality is detected. */
+  readonly modalityDetected: Observable<InputModality>;
+
   /** Emits when the input modality changes. */
-  readonly modalityChanges: Observable<InputModality>;
+  readonly modalityChanged: Observable<InputModality>;
```

Comment:
Why go from "changes" to "changed"?

Reply:
To align with the new `modalityDetected` field (to make them the same tense). WDYT?

Reply:
I don't feel that strongly either way, `ed` is good.